ape_infura/__init__.py

```diff
@@ -2,16 +2,33 @@
 
 from .providers import Infura
 
-NETWORKS = [
-    "mainnet",
-    "ropsten",
-    "rinkeby",
-    "kovan",
-    "goerli",
-]
+NETWORKS = {
+    "ethereum": [
+        "mainnet",
+        "ropsten",
+        "rinkeby",
+        "kovan",
+        "goerli",
+    ],
+    "arbitrum": [
+        "mainnet",
+        "rinkeby",
+        "goerli"
```

Comment:
This is a bug with `ape-arbitrum`, but we only have support for `rinkeby` right now (but it's just called `testnet` which is the bug)
```suggestion
        "testnet",
```

Reply:
is it defined here? https://github.com/ApeWorX/ape-arbitrum/blob/main/ape_arbitrum/ecosystem.py#L9
may be I can just fix it there? or are there other dependencies using testnet instead of rinkeby and goerli?

Reply:
ultimately, it would be best to update it there, however that is a larger breaking change because we will have to make people aware of the update in case they were using it, and also update other plugins too

Reply:
oh, I see
I think the best is to make it with testnet here at first.

And I guess, Iwould make separate requests to ape-arbitrum and ape-infura fixing the names (we can also add backward-compatilibity, should be easy to do). and leave those requests ready for announcements. does it sound good?

Reply:
exactly, make it work with `arbitrum:testnet` here for now, and we'll make the update when fixing https://github.com/ApeWorX/ape-arbitrum/issues/3 at a later point in this plugin

Reply:
I think it's good to add the tests, you can add installing those plugins under this extra and it will get installed during testing:
https://github.com/ApeWorX/ape-infura/blob/main/setup.py#L6

Reply:
ok, would follow
just wasnt sure whether you ok with tests being dependent on the other libraries
thanks for the help!

Reply:
should be good now. thanks a lot for reviewing!